internal/store/installation.go

```diff
@@ -194,6 +194,11 @@ func (sqlStore *SQLStore) CreateInstallation(installation *model.Installation) e
 	installation.ID = model.NewID()
 	installation.CreateAt = GetMillis()
 
+	// add installation ID to mattermost-server env variables
```

Comment:
Although this implementation would definitely achieve the goal of getting an env var set with the installation ID, I think it is probably better suited as hardcoded logic when the installation is created in terms of the custom resource: https://github.com/mattermost/mattermost-cloud/blob/master/internal/provisioner/kops_provisioner_cluster_installation.go#L83

The reasoning is that would help protect against the value being accidentally removed or changed when compared to adding it like standard env configuration here.